README.md

```diff
@@ -12,4 +12,36 @@ _Inspried by two great works:_
 - [1st sample](http://kimsk.github.io/fsreveal-sample-fsx/FsReveal.html#/) generated from [FsReveal.fsx](https://github.com/kimsk/fsreveal-sample-fsx/blob/master/slides/FsReveal.fsx)
 - [2nd sample](http://kimsk.github.io/fsreveal-sample-md/FsReveal.html#/) generated from [FsReveal.md](https://github.com/kimsk/fsreveal-sample-md/blob/master/slides/FsReveal.md) ([raw](https://raw.githubusercontent.com/kimsk/fsreveal-sample-md/master/slides/FsReveal.md))
 
+### Dev
+
+```
+pyenv local 2.7.10
+# npm install -g yarn
+yarn install --ignore-engines
+yarn run build
+# when you wanna test your changes:
+./build.sh GenerateSlides
+```
+
+Source code is in `./src/FsReveal`. Built with `FAKE` and `yarn` (in turn uses `webpack`).
+
+Source code is **also** in `./docs/tools` for the generation part.
+
+#### How it all fits together
+
+The node modules take care of bundling all the JS dependencies. You don't need
+anything other than Yarn to compile, but some of them are dependent on having a
+binary called 'python2' on your PATH.
```

Comment:
I don't know. JS pulls in the world and I'm using the "standard" setup for css/html.